docs/getting-started.md

```diff
@@ -91,192 +73,103 @@ Once the system is verified you will see:
+Next fill out the form with the required values:
 
-Type the following code into the page at the URL above: ABCD-1234
+- Name: **podinfo-deploy**
+- Kubernetes Namespace: **wego-system**  
+  -  **Leave as default**, this is where the automation objects for the application will be created.
+- Source Repo URL: (**your-forked-podinfo-repo**)
+  - For example: `https://github.com/sympatheticmoose/podinfo-deploy` or `git@github.com:sympatheticmoose/podinfo-deploy`
+- Config Repo URL: (**leave blank**)  
+  - It is **recommended** to use a separate repository as your config repo when deploying multiple applications, potentially to multiple clusters, however, for simplicity - you can also use the same repo as the `Source Repo URL`.
+- Path: **./**  
+  - This allows you to specify a particular folder with a repository, should the repo contain more than a single applications deployment manifests.
+- Branch: (**should not require changing**)
 
-Waiting for authentication flow completion...
+Click **Submit** in the bottom right of the form. This will result in an error as we have not recently authenticated with GitHub, so we are not yet authorized to raise a pull request against your repository.
 
-```
+Click **Authenticate with GitHub** within the error message as shown below:
 
-Copy the code and visit the URL shown to grant temporary `repo` access for Weave GitOps.
+![Not authenticated error](/img/dashboard-add-application-auth-error.png)
 
+You will be prompted with a screen to copy a code from, before visiting GitHub to authorize Weave GitOps to write to your repositories. Copy the code to your clipboard and click **AUTHORIZE GITHUB ACCESS**.
+
+Paste the code into the screen as shown below:
 ![device flow activation](/img/github-device-flow-start.png)
 
 You should then see this confirmation:
 
 ![device flow complete](/img/github-device-flow-complete.png)
 
-Once complete, the process will continue and you will see something like:
+Once authorization has completed, navigate back to the GitOps Dashboard.
 
-```console
-Authentication successful!
-
-uploading deploy key
-Deploy key generated and uploaded to git provider
-Adding application:
-
-Name: nginx
-URL: ssh://git@github.com/pzfreo/podinfo-deploy.git
-Path: ./
-Branch: main
-Type: kustomize
-
-✚ Generating Source manifest
-✚ Generating GitOps automation manifests
-✚ Generating Application spec manifest
-► Cloning ssh://git@github.com/pzfreo/podinfo-deploy.git
-Pull Request created: <link to PR>
-
-► Applying manifests to the cluster
-► Committing and pushing gitops updates for application
-✔ App is up to date
-```
+The screen should update with a message that the Pull Request has been created, click **Open Pull Requests** to review the PR in GitHub.
 
-8. A Pull Request has been created against the git branch.
+![Pull request raised](/img/dashboard-add-application-PR-raised.png)
 
-![PR](/img/podinfo-pr.png)
+The Pull Request adds three files which will be deployed to your cluster:
```

Comment:
This is no longer accurate: 4 files (app.yaml, GitRepository, Kustomization, and `kustomize.yaml`) are added in the `apps/<appname>` directory and the user kustomization file for the cluster gets augmented with a reference to the new app directory. the `kustomize.yaml` file in the app directory looks like:
```
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
metadata:
  name: microservices-demo
  namespace: wego-system
resources:
- app.yaml
- microservices-demo-gitops-deploy.yaml
- microservices-demo-gitops-source.yaml
```

Reply:
Noted and will address in the future PR :+1: